src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -30,6 +30,8 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc((MvcOptions options) =>
             {
+                options.Filters.Add(new ProducesAttribute("application/json"));
```

Comment:
>ProducesAttribute [](start = 40, length = 17)

What is the effect here out of curiosity?

Reply:
This basically says that one of the supported output formats of the api is JSON. Values returned from the api will be formatted using JSON by default, unless the action result returns an overriding content type, for example, the ones that return file streams will have "application/octet-stream". I wanted the host of the api (namely dotnet-monitor) to set what the default output format was instead of enforcing it on the controller or each action.